.gitignore

```diff
@@ -48,6 +48,7 @@ install/kubernetes/istio-sidecar-injector-configmap-release.yaml
 install/kubernetes/istio-sidecar-injector.yaml
 install/kubernetes/istio.yaml
 install/kubernetes/istio-all.yaml
+install/kubernetes/sidecar-injector.yaml
```

Comment:
Why do we generate this file ? With helm it's on or off in the generate yaml - no separate install step.

Reply:
For non helm environment (GCP deployment manager ), if we don't have a separate sidecar-injector.yaml, we need to generate 4 yamls: istio.yaml/istio-auth.yaml with/without sidecar injection enabled. I can also do that if it is OK.

Reply:
@yusuoh not familiar with GCP deployment manager, does it instantiate manifests in some way?

Reply:
It is done here:
https://github.com/istio/istio/blob/master/install/gcp/deployment_manager/istio-cluster.jinja#L93

Reply:
cc @selmanj